### ISSUE TITLE
[Feature Request] Support for RTP end point

### ISSUE THREAD

**Issue by kirantpatil** [ant-media/Ant-Media-Server#610]
# Issue
Need Support for RTP end point.

### Short description
I am testing with TI CC3200 module which comes with RTSP protocol and works locally to send video and audio stream but not to cloud.

For which I am planning to skip all RTSP part and use only RTP to stream data(audio/video) directly to the cloud end point(Ant-Media-Server) from my house.

That is the reason I am asking for only RTP endpoint for streaming my home to Ant-Media-Server.

### Environment

* TI CC3200
* Ant Media Server version: All

**Comment by davutcavdar** [comment]
@kirantpatil Ant Media Server, in fact, can fetch RTSP streams directly, you do not need to RTP configuration.

**Comment by kirantpatil** [comment]
@davutcavdar, Yes I know it can fetch RTSP streams but to fetch to cloud we have to either setup static ip or forward stream using router which I am not interested.

I would like to push to cloud without any network setup, for which I am bypassing RTSP and using only direct RTP streaming.

**Comment by SelimEmre** [comment]
Hi @kirantpatil,
Thank you for your feedback! We will consider this feature in new versions. If you need further assistance, please feel free to send an email to contact[at]antmedia.io.

Best Regards,
Selim

**Comment by kirantpatil** [comment]
Dear @SelimEmre, 

Any updates regarding this feature request ?

Thanks.

**Comment by kirantpatil** [comment]
Also we are planning to go for Enterprise license, if it meets our requirements as below.

1. Support RTP endpoint for multiple clients source

2. Some kind of network security mechanism between
    RTP Source <=> Ant-Media-Server <=> End user 

3. Support bidirectional Audio/Video transfer
                         A/V                                A/V (Audio/Video)
    RTP source <==> Ant Media Server <==> End user

4. RTP source is supported by SDP files as mentioned below link,
    https://www.wowza.com/docs/how-to-set-up-live-streaming-using-a-native-rtp-encoder-with-sdp-file

5. Support for A/V recording for each client

**Comment by USAMAWIZARD** [comment]
@kirantpatil I have created a plugin through which you can have RTP output
